### PR TITLE
feature: adding honeycomb datastore support

### PR DESCRIPTION
**Pull request by xoscar**
This PR adds support for Honeycomb as data store

## Changes

- Adds honeycomb support for both BE, FE and CLI
- Adds recipe page for honeycomb
- Adds tracetest x honeycomb example

## Fixes

- #1945 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

https://www.loom.com/share/e6eb3ffdc0a0490792e8220dac29bff2

### CODE REVIEW

File: docs/docs/examples-tutorials/recipes/running-tracetest-with-honeycomb.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+  # OTLP for Tracetest
+  otlp/tt:
+    endpoint: tracetest:4317 # Send traces to Tracetest. Read more in docs here:  https://docs.tracetest.io/configuration/connecting-to-data-stores/opentelemetry-collector
+    tls:
+      insecure: true
+  # OTLP for Honeycomb
+  otlp/hc:
+    endpoint: "api.honeycomb.io:443"
+    headers:
+      "x-honeycomb-team": <HONEYCOMB_API_KEY>
```
Comment (jorgeepc) [inline docs/docs/examples-tutorials/recipes/running-tracetest-with-honeycomb.md:249]: Should we add the api key in the notion page @danielbdias created? https://www.notion.so/kubeshop/Vendor-keys-db37b8db460345b6be3ca4d3c4345940

File: examples/tracetest-honeycomb/readme.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Node.js app with OpenTelemetry, Honeycomb and Tracetest
+
+> [Read the detailed recipe for setting up OpenTelemetry Collector for Honeycomb and Tractest in our documentation.](https://docs.tracetest.io/examples-tutorials/recipes/running-tracetest-without-a-trace-data-store)
```
Comment (jorgeepc) [inline examples/tracetest-honeycomb/readme.md:3]:
```suggestion
> [Read the detailed recipe for setting up OpenTelemetry Collector for Honeycomb and Tracetest in our documentation.](https://docs.tracetest.io/examples-tutorials/recipes/running-tracetest-without-a-trace-data-store)
```

**Review by jorgeepc — APPROVED** [review]
Great job @xoscar, added a couple of comments

File: docs/docs/examples-tutorials/recipes/running-tracetest-with-honeycomb.md (hunk 1 above)
Comment (xoscar) [reply docs/docs/examples-tutorials/recipes/running-tracetest-with-honeycomb.md:249]: Yep

**Pull request merged**